### ISSUE TITLE
Support autocomplete in non-svg files like Vue

### ISSUE THREAD

**Issue by cvharris** [lishu/vscode-svg2#22]
### Steps to Reproduce

1. Create a Vue file with Vue syntax highlighting (*.vue)
2. Create a `<template>` and add any SVG element like `<svg>` or `<circle>`

### What happens

The autocomplete does not work

### What I expect to happen

Autocomplete that happens in *.svg files should work here.

Probably should allow this extension to be configurable to provide autocomplete for any other HTML syntax, including .html, .php, .pug, .jsx, etc.

**Comment by lishu** [comment]
Good idea, I did think about it very early, but because embedding in other languages takes a lot of time to implement, my personal time is limited (mainly poor), and there will be no plans to support embedding in other languages in the short term.

**Comment by FractalHQ** [comment]
Is there support for HTML files yet?  If so, that should work out of the box for `.svelte` files :)
